query-connector/src/app/ui/components/header/header.tsx

```diff
@@ -124,41 +124,51 @@ const HeaderComponent: React.FC<{ authDisabled: boolean }> = ({
             )}
           </div>
         </div>
+        {showMenu && (
+          <div ref={menuRef} className={styles.menuDropdownContainer}>
+            <ul
+              id="dropdown-menu"
+              className={classNames(`usa-nav__submenu`, styles.menuDropdown)}
+            >
+              {/* TODO: Enable this once we can show/hide rules based on actual auth status */}
+              {/* {isProduction && ( */}
+              <>
+                <li className={styles.subMenuItem}>
+                  <Link className={styles.menuItem} href={"/queryBuilding"}>
+                    My queries
+                  </Link>
+                </li>
+                <li className={styles.subMenuItem}>
+                  <Link className={styles.menuItem} href={"/fhir-servers"}>
+                    FHIR Servers
+                  </Link>
+                </li>
+                <li className={styles.subMenuItem}>
+                  <Link className={styles.menuItem} href={"/userManagement"}>
+                    User Management
+                  </Link>
+                </li>
+                <li className={styles.subMenuItem}>
+                  <Link className={styles.menuItem} href={landingPage}>
+                    Log out
+                  </Link>
+                </li>
+              </>
+              {/* )} */}
+            </ul>
+          </div>
+        )}
       </header>
-
-      {showMenu && (
-        <div ref={menuRef} className={styles.menuDropdownContainer}>
-          <ul
-            id="dropdown-menu"
-            className={`usa-nav__submenu ${styles.menuDropdown}`}
-          >
-            {/* TODO: Enable this once we can show/hide rules based on actual auth status */}
-            {/* {isProduction && ( */}
-            <>
-              <li className={styles.subMenuItem}>
-                <Link className={styles.menuItem} href={"/queryBuilding"}>
-                  My queries
-                </Link>
-              </li>
-              <li className={styles.subMenuItem}>
-                <Link className={styles.menuItem} href={"/fhir-servers"}>
-                  FHIR Servers
-                </Link>
-              </li>
-              <li className={styles.subMenuItem}>
-                <Link className={styles.menuItem} href={landingPage}>
-                  Log out
-                </Link>
-              </li>
-            </>
-            {/* )} */}
-          </ul>
-        </div>
-      )}
     </div>
   );
 };
 
-const LOGGED_IN_PATHS = ["/query", "/queryBuilding", "/fhir-servers"];
+const LOGGED_IN_PATHS = [
+  "/query",
+  "/queryBuilding",
+  "/fhir-servers",
```

Comment:
Worth a followup PR: might want to swap `fhir-servers` --> `fhirServers` so that these are consistent.

Reply:
Ticket here! https://linear.app/skylight-cdc/issue/QUE-163/update-fhir-server-paths-to-fhirserver